code/game/area/areas/centcom.dm

```diff
@@ -1,14 +1,5 @@
 
 // CENTCOM
-
-/*
-Side note, be sure to change the network_root_id of any areas that are not a part of centcom
-and just using the z space as safe harbor.  It shouldn't matter much as centcom z is isolated
-from everything anyway.
-
-The areas used here are STRICTLY on the CC Z level.
-*/
-
```

Comment:
why remove the second bit of this

Reply:
I didn't think it was necessary & it's not true anymore.

The file name is ``centcom.dm`` and all areas are subtypes of ``/area/centcom``, but some areas like nukie outpost are lazyloaded and not even on centcom's z level

Reply:
@san7890 hey I heard you liked categorizing areas